Git-Squash.md

```diff
@@ -0,0 +1,42 @@
+# Squashing Multiple Commits Into One
+
+This is an awesome feature of `rebase` that can be used in the `interactive` mode. To squash the last _n_ commits into one, run the following command:
+
+```sh
+git rebase -i HEAD~n
+```
+
+That will open up a text-editor with something similar to the following:
+
+```text
+pick commit_1
+pick commit_2
+pick commit_3
+...
+pick commit_n
+
+# Rebase hash1..hashn onto hash1 (n command(s))
+#
+# Commands:
+# p, pick = use commit
+# r, reword = use commit, but edit the commit message
+# e, edit = use commit, but stop for amending
+# s, squash = use commit, but meld into previous commit
+# f, fixup = like "squash", but discard this commit's log message
+# x, exec = run command (the rest of the line) using shell
+# d, drop = remove commit
+#
+# These lines can be re-ordered; they are executed from top to bottom.
+#
+# If you remove a line here THAT COMMIT WILL BE LOST.
+#
+# However, if you remove everything, the rebase will be aborted.
+#
+# Note that empty commits are commented out
+```
```

Comment:
@Rafase282 I think adding instructions of the editor is too much.

Lets just say. "Also, you may read the instructions in the text editor"